src/styles/images/index.md.njk

```diff
@@ -28,6 +28,8 @@ If your image represents something physical, such as a letter, document or credi
 
 ## Alternative text
 
+This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
```

Comment:
@calvin-lau-sig7 What do you reckon? :)

Reply:
Looks good! Could we be a bit more specific about what GOV.UK content is? 

Could say 'mainstream content' or perhaps say guidance, but we'd have to rewrite the sentence so we don't end up saying it twice to describe different things.

```suggestion
This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
```

Reply:
@calvin-lau-sig7 That's a nice suggestion, thanks!